src/patterns/exit-a-page-quickly/index.md

```diff
@@ -0,0 +1,104 @@
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This component is currently experimental. You'll need to do your own research to decide whether to use this pattern and add the <a class="govuk-link" href="/components/exit-this-page/">Exit this page</a> component to your service.
+---
+
+{% from "_example.njk" import example %}
+
+Give users a way to quickly exit a service, website or application quickly and safely.
+Use this pattern to add the [Exit this page](/components/exit-this-page/) component to your service and help users use it safely to protect their privacy.
+
+## When to use this pattern
+
+Use this pattern to help the user protect their privacy when your service contains sensitive information that could:
+- put someone at risk of abuse or retaliation
+- reveal someone’s plans to avoid or escape from harm
+
+For example, a potential victim using a service to help them leave domestic abuse.
+
+Other situations and topics where sensitive information could put users at risk might include:
+- rape or sexual assault
+- child abuse and neglect
+- any other type of abuse
+- stalking and harassment
+- reporting crime or fraud (whistleblowing)
+
+## When not to use this pattern
+
+You should not use this pattern for standalone content pages, such as dashboards and guidance. In these cases, use the Exit this page component on its own.
+
+## How it works
+
+First, you’ll need to add the [Exit this page](/components/exit-this-page/) component to your service and decide how it should work within your service.
+
+The component has several parts, including:
+- a button, that takes the user to another website
+- a secondary link, to give people that use assistive technologies a way to activate the component
+- a loading overlay, to immediately clears any content off the browser until the next website loads
+
+Next, you’ll need to create new content pages in your service.
+
+The purpose of these pages are to:
+- tell the users how to use ‘Exit this page’ to protect their privacy
+- give users information they need to stay safe online
+
+Keep in mind that this pattern is not a complete solution to eliminating all possible risk to the user. Perpetrators can monitor potential victims through other methods, such as malicious software.
+
+## Interruption page
+
+Create a page to tell users what Exit this page does.
+
+Show this page just after the start point of your service, and just before the page where the user will see the Exit this page button for the first time. 
+
+On longer services, you might need more than one interruption page.
+
+The page should tell the user:
+- about the ‘Exit this page’ button and what it’s for
+- what happens when they press it
+- that they can also activate Exit this page by pressing shift 3 times or [using the secondary link](/exit-this-page/#adding-the-secondary-link)
+
+The page should also tell the user that:
+- their browser history will not be erased, which can still put them at risk
```

Comment:
Good idea, it's more for end users (rather than the service teams guidance here), but we could use the term here to make sure it's carried through!

```suggestion
- their internet browser history will not be erased, which can still put them at risk
```

Reply:
"Internet browsing history"